templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -149,7 +149,7 @@
 							<div class="ui buttons two fluid start-add">
 								<button onclick="this.disabled=true;toggleStopwatch()" class="ui button poping up start" data-content='{{.i18n.Tr "repo.issues.start_tracking"}}' data-position="top center" data-variation="small inverted">{{.i18n.Tr "repo.issues.start_tracking_short"}}</button>
 								<button onclick="timeAddManual()" class="ui button green poping up add-time" data-content='{{.i18n.Tr "repo.issues.add_time"}}' data-position="top center" data-variation="small inverted">{{.i18n.Tr "repo.issues.add_time_short"}}</button>
-								<div class="ui mini modal">
+								<div class="ui mini modal time-add-manual">
```

Comment:
Why was this changed here?

Reply:
Probably because of a merging issue with githubs master caused by my editor...